content/en/xlt/load-testing/How-Tos/debug-data-result-browser.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "How to Add Debug Data to the Result Browser"
+linkTitle: "Debug Data in Result Browser"
+
+weight: 260
+type: docs
+
+description: >
+  A helpful feature for reconstructing testcase failures.
+---
+
+To aid in error analysis, XTC provides a feature that allows you to add your custom debug data to the result browser: the session's **value log** is a storage for session-specific test parameters and result data. Any value you add to this log will later be available in the result browser. 
+
+{{< image src="how-to/custom_result_browser_data.png" >}}
+Result browser containing custom debug data.
+{{< /image >}}
+
+Note that the log will be cleared with each new iteration of the test scenario.This data may help you to reconstruct and rerun a failed test case iteration without having to dig into log files. This is especially useful if your test case uses random or randomly chosen test parameters.
```

Comment:
Space missing before "This data may"
Suggested change of that stentence:
The data may help to reproduce test failures by rerunning test scenario with the same data more easily, or simply present contextual data for recreation of error situations, either manually or automatically.